build.sbt

```diff
@@ -2,8 +2,9 @@ lazy val root = (project in file("."))
   .settings(
     name := "uknw-auth-checker-api-tests",
     version := "0.1.0",
-    scalaVersion := "2.13.12",
-    libraryDependencies ++= Dependencies.test
+    scalaVersion := "3.3.3",
+    libraryDependencies ++= Dependencies.test,
+    (Compile / compile) := ((Compile / compile) dependsOn (Compile / scalafmtSbtCheck, Compile / scalafmtCheckAll)).value
```

Comment:
This looks like a merge conflict. The compile/compile line was removed in a previous PR.